External/Plugins/HaXeContext/Completion/CodeGenerator.cs

```diff
@@ -10,13 +10,18 @@
 
 namespace HaXeContext.Completion
 {
-    internal class CodeGenerator : IContextualGenerator
+    internal class CodeGenerator : ASGenerator
     {
-        public bool ContextualGenerator(ScintillaControl sci, List<ICompletionListItem> options, ASResult expr)
+        public override bool ContextualGenerator(ScintillaControl sci, List<ICompletionListItem> options, ASResult expr)
         {
-            if ((ASContext.Context.CurrentClass.Flags & FlagType.Interface) != 0
```

Comment:
Interface handling isn't needed anymore?

Reply:
Thanks, fixed. I will write test for it today.